gointelowl/me.go

```diff
@@ -0,0 +1,214 @@
+package gointelowl
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+type Details struct {
+	Username  string `json:"username"`
+	FirstName string `json:"first_name"`
+	LastName  string `json:"last_name"`
+	FullName  string `json:"full_name"`
+	Email     string `json:"email"`
+}
+
+type AccessDetails struct {
+	TotalSubmissions int `json:"total_submissions"`
+	MonthSubmissions int `json:"month_submissions"`
+}
+
+type User struct {
+	User   Details       `json:"user"`
+	Access AccessDetails `json:"access"`
+}
+
+type UserService struct {
+	client *IntelOwlClient
+}
+
+type Owner struct {
+	Username string    `json:"username"`
+	FullName string    `json:"full_name"`
+	Joined   time.Time `json:"joined"`
+}
+
+type Organization struct {
+	MembersCount int        `json:"members_count"`
+	Owner        Owner      `json:"owner"`
+	IsUserOwner  bool       `json:"is_user_owner,omitempty"`
+	CreatedAt    *time.Time `json:"created_at,omitempty"`
+	Name         string     `json:"name"`
+}
+
+type OrganizationParams struct {
+	Name string `json:"name"`
+}
+
+type MemberParams struct {
+	Username string `json:"username"`
+}
+
+type Invite struct {
+	Id        int       `json:"id"`
+	CreatedAt time.Time `json:"created_at"`
+	Status    string    `json:"status"`
+}
+
+type Invitation struct {
+	Invite
+	Organization Organization `json:"organization"`
+}
+
+type InvitationParams struct {
+	Organization OrganizationParams `json:"organization"`
+	Status       string             `json:"status"`
+}
+
+// Get retrieves user details
+//
+//	Endpoint: GET /api/me/access
+//
+// IntelOwl REST API docs: https://intelowl.readthedocs.io/en/latest/Redoc.html#tag/me/operation/me_access_retrieve
+func (userService *UserService) Access(ctx context.Context) (*User, error) {
+	requestUrl := fmt.Sprintf("%s/api/me/access", userService.client.options.Url)
```

Comment:
Let's add these routes to [constants.go](https://github.com/intelowlproject/go-intelowl/blob/main/gointelowl/constants.go) as well!

Reply:
@sp35 I planned on adding and revamping the constants.go file after the merge because I wanted to use the file in the unit tests as well!

Reply:
can I do revamping in this PR? @sp35

Reply:
Let's do in a separate PR then